sdks/java/io/kinesis/build.gradle

```diff
@@ -17,7 +17,7 @@
  */
 
 apply from: project(":").file("build_rules.gradle")
-applyJavaNature(failOnWarning: true)
+applyJavaNature(failOnWarning: true, shadowClosure: {})
```

Comment:
The default if nothing is included and nothing is excluded means to include everything which is not what I think we want.

Reply:
Looks like this is unnecessary now. Dropped.